### DISCUSSION TITLE
Inquiry about how to get column height

### DISCUSSION THREAD

**Discussion by siyingch86** [patrykandpatrick/vico#656] [Questions]
Hi, I want to get the bar height(column height) in the stacked bar chart. Is there any public API to access that? Thanks

**Comment by patrickmichalik** [comment]
Hello! Could you elaborate on what you’re looking to do? 

- Are you referring to the total height in pixels or the aggregate _y_ value?
- What will the value in question be used for?

**Reply by siyingch86** [reply]
To answer your questions: 
1. the total height in pixels 
2. to adjust the bar shape as needed

**Reply by Gowsky** [reply]
Do you mind explaining precisely what you are looking to achieve? It sounds like it's related to #633 which already has been answered. We need to know the specifics of your problem to suggest a good solution. How is the shape supposed to change?

You can access the column height with `ColumnCartesianLayer.stackInfo`, but you will need to subclass `ColumnCartesianLayer` in order to access the field.
